source/declarations.tex

```diff
@@ -2935,8 +2935,12 @@
   using C::x;
   int x(double);
 };
-int f(D* d) {
-  return d->x();    // ambiguous: \tcode{B::x} or \tcode{C::x}
+
+void f(D* d) {
+  d->x();    // ambiguous: while member lookup of \tcode{x} in implicit naming class \tcode{D} is unambiguous, the selected 
```

Comment:
Yes - the using declaration simply helps to guide member lookup so that x() is not hidden.